src/components/RadioGroup/index.examples.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import {storiesOf} from '@kadira/storybook'
+import RadioGroup from './index'
+
+const decoratorClasses = 'flex flex-column content-center justify-around flex-wrap items-center vh-100 bg-navy'
```

Comment:
Seems you are using this in almost all of the examples; could pull it out into a module and import it or maybe there is a way to declare a global decorator for all stories in the storybook config